doc/nrf/releases/release-notes-latest.rst

```diff
@@ -362,10 +386,6 @@ The following list summarizes the most important changes inherited from upstream
     * Added an API for resetting a node (:c:func:`bt_mesh_cfg_node_reset`).
     * Added an API for setting network transmit parameters (:c:func:`bt_mesh_cfg_net_transmit_set`).
 
-* Trusted Firmware-M:
```

Comment:
Has that moved?

Reply:
yes, together with the other zephyr modules.